package.json

```diff
@@ -29,7 +29,7 @@
     "geocoder-arcgis": "^2.0.5",
     "next": "^11.1.1",
     "next-compose-plugins": "^2.2.1",
-    "next-i18next": "^8.4.0",
+    "next-i18next": "8.4.0",
```

Comment:
This is the bugfix for the language switch not working. I used a fixed version for this module and not allowing to use any higher versions where our code does not work anymore.